java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -1241,6 +1242,43 @@ public T zcard(@NonNull String key) {
         return getThis();
     }
 
+    /**
+     * Removes and returns up to <code>count</code> members with the lowest scores from the sorted set
```

Comment:
same as above please

Reply:
Bit confused as to what you are referring to.

Reply:
Make it match: SortedSetBaseCommands.java documentation